### PR TITLE
[WIP] Convert to IETF Internet Draft format

### PR DESCRIPTION
**Pull request by joho**
It's time to address #184 and take this puppy to the big leagues.

- [x] get into a legit internet draft format
- [ ] replace meme driven section with "culture/meritocracy" (in progress)
- [ ] fill it with more fancy IETF words and stuff

Also adds a licence and a makefile.

### CODE REVIEW

File: 7xx.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  workgroup = ""
+  keyword = ["http", "errors"]
+
+  date = 2017-04-01T00:00:00Z
+
+  [[author]]
+  initials="J. R."
+  surname="Barton"
+  fullname="J. R. Barton"
+  organization = "Railscamp"
```
Comment (dgoodlad) [inline 7xx.md]: :camping:

**Comment by cmckni3** [comment]
I propose "Fucking semi-colons"

**Comment by stendarr** [comment]
@cmckni3 Too many or not enough semi-colons? Oh no, wait! We already have that sorted out. See 754 and 755.

**Comment by cmckni3** [comment]
Not enough semi-colons or syntax error technically in languages that require semi-colons.

I like saying yelling "Fucking semi-colons" when I miss a semi-colon.

**Comment by extensibleseth** [comment]
Some of the memes are just too good and have value as historical context. While meritocracy has value, and space is limited, ponies will be ever confounding. Another way of looking at it is that this shapes the shibboleths of the future and can impart relevancy.

File: 7xx.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+{mainmatter}
+
+# Introduction
+
+Some words and stuff. Reference to [@?RFC7231], looking to classify 500 series as unknown unknowns, and the 700 series as known unknowns.
+
+## Terminology
+
+The keywords **MUST**, **MUST NOT**, **REQUIRED**, **SHALL**, **SHALL NOT**, **SHOULD**,
```
Comment (sr229) [inline 7xx.md]: The Recursion here is godlike

**Comment by joho** [comment]
OK I never really finished this branch, but I'm going to merge as is to master to save me from rebasing hell.

**Pull request merged**